api/utils/request.js

```diff
@@ -25,7 +25,7 @@ class RequestUtils {
      * @param req Request
      */
     getPagingParams(req) {
-        const limit = req.query.limit || 100;
+        const limit = req.query.limit || 500;
```

Comment:
So I noticed that we were not returning all the locations hits in the API response. The API is designed to receive the `limit` as a parameter in request, although that param is not being passed in the calls coming from UI the api is defaulting to 100 and not returning all locations that were queried.  hence locations like REEB was not showing up. As a workaround increasing it to 500, although @mihiramin89 we need to fix it in frontend.
for example - 
current state (return only 100 results as per default) -
`http://localhost:3000/api/v2/location?taxonomyId=5&agencyId=11110,30525,30525,13800,29900,23560,23550,28500,28500,40500,39650,39100,7800,7700,5120,4124006,5590927,8700,100,5300,11110,45350,30232,24300,50100,41400,26200,7800,40850,48325,50025,50085,7880,4549055,45300,39650,10800,101478,16000,19700,23560,1164300,29900,1610753,48325,46570,2950,10775,19700,36100,29900,29800,1013310,23560,19700,922790,39100,13800,50350,50350,10800,2950,23560,7800,48325,30232,4781935,4200,23560,35325,45350,46575,6392008,5300,2157603,2157603,39650,26200,13800,50025,45350,50025,2157603,6009842,15500,5145,5250,7800,1164300,26200,3974916,4873121,4132490,4132490,4149423,29800,5913514,5913236,5388949,5750620`
vs  below fetches 269 records.
`http://localhost:3000/api/v2/location?taxonomyId=5&agencyId=11110,30525,30525,13800,29900,23560,23550,28500,28500,40500,39650,39100,7800,7700,5120,4124006,5590927,8700,100,5300,11110,45350,30232,24300,50100,41400,26200,7800,40850,48325,50025,50085,7880,4549055,45300,39650,10800,101478,16000,19700,23560,1164300,29900,1610753,48325,46570,2950,10775,19700,36100,29900,29800,1013310,23560,19700,922790,39100,13800,50350,50350,10800,2950,23560,7800,48325,30232,4781935,4200,23560,35325,45350,46575,6392008,5300,2157603,2157603,39650,26200,13800,50025,45350,50025,2157603,6009842,15500,5145,5250,7800,1164300,26200,3974916,4873121,4132490,4132490,4149423,29800,5913514,5913236,5388949,5750620&limit=1000`